rllib/algorithms/marwil/marwil.py

```diff
@@ -248,6 +321,81 @@ def get_default_policy_class(
 
     @override(Algorithm)
     def training_step(self) -> ResultDict:
+        if self.config.enable_env_runner_and_connector_v2:
+            return self._training_step_new_stack()
+        elif self.config.enable_rl_module_and_learner:
```

Comment:
Let's retire the hybrid stack for BC and MARWIL altogether ...

Reply:
But before we do this: Isn't Riot using the hybrid stack with BC?

Reply:
That one is gone now.